### PR TITLE
Community Invitations/Pending Memberships

### PR DESCRIPTION
**Pull request by me-andre**
### Describe the background of your pull request

What does your pull request do? Does it solve a bug (which one?), add a feature?

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/alkem-io/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/alkem-io/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/alkem-io/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

File: src/domain/challenge/space/pages/SpaceCommunity/useSpaceCommunityContext.ts [hunk 1]
```diff
@@ -62,6 +67,14 @@ const useSpaceCommunityContext = (spaceId: string) => {
     ),
   };
 
+  const { data: spaceDisplayNameData } = useSpaceDisplayNameQuery({
```
Comment (ccanos) [inline src/domain/challenge/space/pages/SpaceCommunity/useSpaceCommunityContext.ts:70]: Another query just for the displayName sounds excesive to me, isn't there a better way to obtain it or isn't it easy to pass it from higher... in SpaceCommunityPage you have a useSpace which is not nice, but has the profile.

File: src/domain/challenge/space/pages/SpaceCommunity/spaceApplicationsInvitations.graphql [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+}
+
+fragment AdminSpaceCommunityCandidateMember on User {
+  id
+  nameID
+  email
+  profile {
+    id
+    displayName
+    visual(type: AVATAR) {
```
Comment (ccanos) [inline src/domain/challenge/space/pages/SpaceCommunity/spaceApplicationsInvitations.graphql:71]: if it's an easy change, I'm trying to do `avatar: visual....`
If you need to refactor something else forget it.

File: src/domain/shared/components/RadioButtons/RadioButton.tsx [hunk 3]
```diff
@@ -1,33 +1,56 @@
...
   disabled?: boolean;
   iconComponent: ComponentType<SvgIconProps>;
   onClick?: (value: Value) => void;
 }
 
 const StyledButton = styled(IconButton)(({ theme }) => ({
-  width: iconButtonSize,
-  height: iconButtonSize,
-  padding: theme.spacing(2),
+  // width: iconButtonSize,
```
Comment (ccanos) [inline src/domain/shared/components/RadioButtons/RadioButton.tsx:18]: should we remove these?

**Review by ccanos — CHANGES_REQUESTED** [review]
A few small changes requested. as we talked most can be ignored if they imply a big refactor

File: src/domain/community/pendingMembership/PendingMemberships.graphql [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+  invitations {
+    ...PendingMembershipInvitation
+  }
+}
+
+fragment PendingMembershipInvitation on Invitation {
+  id
+  welcomeMessage
+  createdBy {
+    profile {
```
Comment (ccanos) [inline src/domain/community/pendingMembership/PendingMemberships.graphql:93]: missing id

**Review by ccanos — CHANGES_REQUESTED** [review]
a few small changes

**Review by ccanos — CHANGES_REQUESTED** [review]
All good with the code, changes look good to me and you have addressed all my comments, but I have tested it locally and I'm getting this dialog that stays after Accepting an Invitation... 
image

Not sure if I'm doing something wrong, the mutation is returning ok:
image
I'm on `develop` branch on the server

**Review by ccanos — APPROVED** [review]

**Review by valentinyanakiev — APPROVED** [review]

**Pull request merged**